go.mod

```diff
@@ -12,8 +12,8 @@ require (
 	k8s.io/apimachinery v0.25.0
 	k8s.io/client-go v0.25.0
 	k8s.io/component-base v0.25.0
-	open-cluster-management.io/addon-framework v0.5.1-0.20221206033210-55b40b1bc8c6
-	open-cluster-management.io/api v0.8.1-0.20220919023232-a2688935edf3
```

Comment:
Were these necessary for a feature?

Reply:
Not necessary, but I saw that we were running a prerelease version and decided to sneak in an update.

I can make this a separate PR.

Reply:
And I *hadn't* noticed we had a pre-release version. The update should be good.

Reply:
The prerelease was to catch a hosted mode annotation update, which had a typo. (You know me and typos!) Update should be fine.